typescript/sdk/src/token/nativeTokenMetadata.ts

```diff
@@ -0,0 +1,25 @@
+import { ProtocolType } from '@hyperlane-xyz/utils';
+
+import { NativeToken } from '../metadata/chainMetadataTypes.js';
+
+export const PROTOCOL_TO_DEFAULT_NATIVE_TOKEN: Record<
+  ProtocolType,
+  NativeToken
+> = {
+  [ProtocolType.Ethereum]: {
```

Comment:
Note: I considered using the data from the registry here instead of duping the values for ethereum, solana, cosmoshub but that would introduce a package import cycle sdk<->registry